crates/algorithms/src/lib.rs

```diff
@@ -10,6 +10,7 @@
 extern crate alloc;
 #[cfg(not(feature = "std"))]
 extern crate core;
+extern crate core;
```

Comment:
Not sure why is there a double line here, and core should only be extern if we don't use std^^